packages/fiori/src/NotificationListItemBase.ts

```diff
@@ -0,0 +1,277 @@
+ *
+ * @abstract
+ * @constructor
+ * @author SAP SE
+ * @alias sap.ui.webc.fiori.NotificationListItemBase
+ * @extends sap.ui.webc.main.ListItemBase
+ * @tagname ui5-li-notification-group
+ * @since 1.0.0-rc.8
+ * @appenddocs sap.ui.webc.fiori.NotificationAction
+ * @public
+ */
+
+/**
+ * Fired when the <code>Close</code> button is pressed.
+ *
+ * @event sap.ui.webc.fiori.NotificationListItemBase#close
+ * @param {HTMLElement} item the closed item.
+ * @public
+ */
+@event("close", {
+	 detail: {
+		item: HTMLElement,
+	},
+})
+class NotificationListItemBase extends ListItemBase {
+/**
+ * Defines the <code>titleText</code> of the item.
+ * @type {string}
+ * @defaultvalue ""
+ * @public
+ * @name sap.ui.webc.fiori.NotificationListItemBase.prototype.titleText
+ */
+@property()
+titleText!: string;
+
+/**
+ * Defines the <code>priority</code> of the item.
+ * Available options are:
+ * <ul>
+ * <li><code>None</code></li>
+ * <li><code>Low</code></li>
+ * <li><code>Medium</code></li>
+ * <li><code>High</code></li>
+ * </ul>
+ * @type {sap.ui.webc.main.types.Priority}
+ * @defaultvalue "None"
+ * @public
+ * @name sap.ui.webc.fiori.NotificationListItemBase.prototype.priority
+ */
+@property({ type: Priority, defaultValue: Priority.None })
+priority!: Priority;
+
+/**
+ * Defines if the <code>close</code> button would be displayed.
+ * @type {boolean}
+ * @defaultvalue false
+ * @public
+ * @name sap.ui.webc.fiori.NotificationListItemBase.prototype.showClose
+ */
+@property({ type: Boolean })
+showClose?: boolean;
+
+/**
+ * Defines if the <code>notification</code> is new or has been already read.
+ * <br><br>
+ * <b>Note:</b> if set to <code>false</code> the <code>titleText</code> has bold font,
+ * if set to true - it has a normal font.
+ * @type {boolean}
+ * @defaultvalue false
+ * @public
+ * @name sap.ui.webc.fiori.NotificationListItemBase.prototype.read
+ */
+@property({ type: Boolean })
+read!: boolean;
+
+/**
+ * Defines if a busy indicator would be displayed over the item.
+ * @type {boolean}
+ * @defaultvalue false
+ * @public
+ * @name sap.ui.webc.fiori.NotificationListItemBase.prototype.busy
+ * @since 1.0.0-rc.8
+ */
+@property({ type: Boolean })
+busy!: boolean;
+
+/**
+ * Defines the delay in milliseconds, after which the busy indicator will show up for this component.
+ *
+ * @type {sap.ui.webc.base.types.Integer}
+ * @defaultValue 1000
+ * @public
+ * @name sap.ui.webc.fiori.NotificationListItemBase.prototype.busyDelay
+ */
+@property({ validator: Integer, defaultValue: 1000 })
+busyDelay!: number;
+
+/**
+ * Defines the actions, displayed in the top-right area.
+ * <br><br>
+ * <b>Note:</b> use the <code>ui5-notification-action</code> component.
+ *
+ * @type {sap.ui.webc.fiori.INotificationAction[]}
+ * @slot
+ * @public
+ * @name sap.ui.webc.fiori.NotificationListItemBase.prototype.actions
+ */
+@slot()
+actions!: Array<HTMLElement & NotificationActionInfo>
```

Comment:
When NotificationAction is migrated, probably this will need update and will look better.